projects/plutus/inc/bps_heartbeat.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+// BPS heartbeat - sends periodic heartbeat CAN messages containing the fault state
+// Requires soft timers, CAN to be initialized
```

Comment:
Should probably mention which boards should subscribe to this?

Reply:
It's set using `storage->expected_bitset`, so theoretically any. :P
I can add a high-level description such as "critical/powertrain boards should respond".